pkg/cloudprovider/azure/block_storage_adapter.go

```diff
@@ -90,12 +91,13 @@ func NewBlockStorageAdapter(location string, apiTimeout time.Duration) (cloudpro
 	disksClient := disk.NewDisksClient(cfg[azureSubscriptionIDKey])
 	snapsClient := disk.NewSnapshotsClient(cfg[azureSubscriptionIDKey])
 
-	disksClient.Authorizer = spt
-	snapsClient.Authorizer = spt
+	authorizer := autorest.NewBearerAuthorizer(&spt.Token)
```

Comment:
this should just be `authorizer := autorest.NewBearerAuthorizer(spt)` -- hit this bug when testing Azure

Reply:
thanks

Reply:
Is it just this 1 line change? Any other Azure changes?

Reply:
tested backup with snapshots, and GC of said backup, with this change -- all looks good